docs/framework/wpf/advanced/xaml-syntax-in-detail.md

```diff
@@ -37,7 +37,7 @@ This topic defines the terms that are used to describe the elements of XAML synt
 ## The XAML Language Specification  
  The XAML syntax terminology defined here is also defined or referenced within the XAML language specification. XAML is a language based on XML and follows or expands upon XML structural rules. Some of the terminology is shared from or is based on the terminology commonly used when describing the XML language or the XML document object model.  
   
```

Comment:
something wrong with the script here.

Reply:
i will check what happened. lets just ignore